src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -207,6 +223,10 @@ export class OrganizationSubscriptionComponent implements OnInit {
         return this.sub.plan.hasAdditionalSeatsOption;
     }
 
+    get isSponsoredSubscription(): boolean {
+        return this.sub.subscription?.items.some(i => i.sponsoredSubscriptionItem);
```

Comment:
I would really like to add some new domain object for this knowledge, but since the rest of the component uses `OrgnaizationSubscriptionResponse` directly I think it's out of scope in this PR, but we should tackle it someday.